<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>595</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnimExport</string>
  </property>
  <widget class="QPushButton" name="BTN_root_joint_select">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>101</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_RootJoint">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>107</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Root Joint :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_export_anim">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>136</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export ANIM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_import_fbx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>367</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Scenes /  Import FBX File</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_RootJointName">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>104</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Action_Start_Frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>38</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LB_ActionStartFrame">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Action Start Frame :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_timeline_set">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>350</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Timeline Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_import_anim">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>350</y>
     <width>152</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear / Import Anim</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>370</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Export_Mov_Path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>493</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="LW_animList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>366</width>
     <height>103</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Preview_File_Open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>93</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Prv Open</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/autodesk/maya2017/icons/mayaHIK/HIKCharacterization_ToolBar_ToggleFigure.png</normaloff>../../../../../../usr/autodesk/maya2017/icons/mayaHIK/HIKCharacterization_ToolBar_ToggleFigure.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Open_Mov_Folder">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>493</y>
     <width>38</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/icons/gnome/32x32/places/folder.png</normaloff>../../../../../../usr/share/icons/gnome/32x32/places/folder.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Mov_Path_Modify">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>493</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Hud">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>396</y>
     <width>54</width>
     <height>87</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HUD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Playblast">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>396</y>
     <width>81</width>
     <height>87</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/icons/gnome/48x48/mimetypes/video-x-generic.png</normaloff>../../../../../../usr/share/icons/gnome/48x48/mimetypes/video-x-generic.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Project_Name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>428</y>
     <width>132</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Project">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>430</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Project : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Char_Name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>396</y>
     <width>132</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>humanType01</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_CharName">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>398</y>
     <width>84</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Char Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_File_Name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>458</y>
     <width>132</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="LB_FileName">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>460</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_PreviewExport">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>310</y>
     <width>369</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>▒▒▒   Preview Export   ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_AnimExport">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>368</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>▒▒▒   Anim Export   ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Open_Anim_Folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>136</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/icons/gnome/32x32/places/folder.png</normaloff>../../../../../../usr/share/icons/gnome/32x32/places/folder.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="LB_mbExport">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>520</y>
     <width>369</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>▒▒▒   Export to Motion Builder   ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_Export_MB">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>555</y>
     <width>328</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select  [ 1.Character + 2.Environment ]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MB_HELP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>555</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
